apps/dataflow/src/scraper/fms/s3_utils.py

```
@@ -0,0 +1,248 @@
+        print(f"Error uploading {local_folder_path}: {e}")
+        return False
+
+
+def upload_generic_file(local_file_path, s3_object_name, file_type="octet-stream"):
+    """Upload a JSON file to S3 bucket
+
+    Args:
+        local_file_path (str): The local object name/path
+        s3_object_name (str): The S3 object name/path
+        file_type (str): the file type
+    """
```

Ruff (0.14.14)

[warning] First line should end with a period

Add period

(D400)